tidb-cloud/troubleshoot-import-permission-error.md

```diff
@@ -0,0 +1,148 @@
+
+If your policy is not correctly configured as the preceding example shows, correct the `Resource` fields in your policy and try importing data again.
+
+> **Tip:**
+>
+> If you have updated the permission policy multiple times and still get the `AccessDenied` error during data import, you can try to revoke active sessions. Go to **IAM** > **Access Management** > **Roles**, find **Revoke active sessions** and click the button to revoke active sessions. Then, retry the data import.
+>
+> Note that this might affect your other applications.
+
+## Check the bucket policy
+
+In the AWS Management Console, open the Amazon S3 console, go to **Bucket** > **Permissions**, and find the **Bucket policy** page. By default, this page has no policy value. If any denied policy is displayed on this page, the `AccessDenied` error might occur during data import.
+
+If you see a denied policy, check whether the policy relates to the current data import. If yes, delete it from the page and retry the data import.
+
+## Check the trust entity
+
+In the AWS Management Console, go to **IAM** > **Access Management** > **Roles**, click the **Trust relationships** tab, and you will see the trusted entities.
+
+The following is a sample trust entity:
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::380838443567:root"
+            },
+            "Action": "sts:AssumeRole",
+            "Condition": {
+                "StringEquals": {
+                    "sts:ExternalId": "696e6672612d617069a79c22fa5740944bf8bb32e4a0c4e3fe"
+                }
+            }
+        }
+    ]
+}
+```
+
+In the sample trust entity:
+
+- `380838443567` is the TiDB Cloud Account ID. Make sure that this field in your trust entity matches your TiDB Cloud Account ID.
+- `696e6672612d617069a79c22fa5740944bf8bb32e4a0c4e3fe` is the TiDB Cloud External ID. Make sure that this field in your trusted entity matches your TiDB Cloud External ID.
+
+## Check the Object Ownership
+
+In the AWS Management Console, open the Amazon S3 console, go to **Bucket** > **Permissions**, and find the **Object Ownership** page. Make sure that the "Object Ownership" configuration is "Bucket owner enforced".
+
+If the configuration is not "Bucket owner enforced", the `AccessDenied` error occurs, because your account does not have enough permissions for all objects in this bucket.
+
+To handle the error, click **Edit** on the upper right corner of the Object Ownership area and change the ownership to "Bucket owner enforced". Note that this might affect your other applications that are using this bucket.
```

Comment:
```suggestion
To handle the error, click **Edit** in the upper-right corner of the Object Ownership area and change the ownership to "Bucket owner enforced". Note that this might affect your other applications that are using this bucket.
```